src/main/resources/addlinks.properties

```diff
@@ -1,9 +1,9 @@
 filterFileRetrievers=true
-executeAsPersonID=8939149
+executeAsPersonID=
```

Comment:
Just curious why the person id was removed?

Reply:
Just because it was hard-coded. All of our past AddLinks releases have been 'done' by Solomon as a result instead of the person running it